packages/common/util.js

```diff
@@ -7,13 +7,14 @@ const os = require('os');
 /**
  * Synchronously makes a temporary directory, smoothing over the differences between
  * mkdtempSync in node.js for various platforms and versions
+ * 
  * @param {string} name - A base name for the temp dir, to be uniquified for the final name
- * @return - The absolute path to the created dir
+ * @returns {string} - The absolute path to the created dir
  */
 exports.mkdtempSync = (name) => {
-  if (fs.mkdtempSync) {
-    return fs.mkdtempSync(`gitc_${name}`);
-  }
+  // if (fs.mkdtempSync) {
```

Comment:
This should be removed if not needed anymore. Also please consider writing tests for this function

Reply:
done